framework/include/kernels/MatDiffusionBase.h

```diff
@@ -52,20 +52,27 @@ class MatDiffusionBase : public DerivativeMaterialInterface<JvarMapKernelInterfa
   const bool _is_coupled;
 
   /// int label for the Concentration
-  unsigned int _conc_var;
+  unsigned int _v_var;
 
   /// Gradient of the concentration
-  const VariableGradient & _grad_conc;
+  const VariableGradient & _grad_v;
 };
 
 template <typename T>
 InputParameters
 MatDiffusionBase<T>::validParams()
 {
   InputParameters params = ::validParams<Kernel>();
-  params.addParam<MaterialPropertyName>("D_name", "D", "The name of the diffusivity");
+  params.addDeprecatedParam<MaterialPropertyName>(
+      "D_name",
+      "The name of the diffusivity",
+      "This parameter has been renamed to 'diffusivity', which is more mnemonic and more conducive "
+      "to passing a number literal");
+  params.addParam<MaterialPropertyName>(
+      "diffusivity", "D", "The diffusivity value or material property");
   params.addCoupledVar("args", "Vector of arguments of the diffusivity");
```

Comment:
maybe, "Optional vector of arguments for the diffusivity. If provided and diffusivity is a derivative parsed material, Jacobian contributions from the diffusivity will be automatically computed"

Reply:
Well, actually this pr isn't really about non-AD...just a suggestion

Reply:
I like that